### ISSUE TITLE
UNUSED_LOCAL_VARIABLE false positive when used in new object initialization

### ISSUE THREAD

**Issue by KamasamaK** [cflint/CFLint#577]
The follow code triggers UNUSED_LOCAL_VARIABLE for `sampleQuery`. It does this regardless of whether it is referenced with named argument or positionally.

```
package query function foo() {
	var sampleQuery = queryNew("valid");
	var sampleQueryOfQuery = new Query(SampleQuery="#sampleQuery#");
	sampleQueryOfQuery.setName("sampleQueryOfQuery");
	sampleQueryOfQuery.setDbType("query");
	sampleQueryOfQuery.setSql("
		SELECT	*
		FROM	SampleQuery
		WHERE	valid = 1
	");

	return sampleQueryOfQuery.execute().getResult();
}
```

**Comment by ryaneberly** [comment]
Drilling down in the contents of new() was suppressed in https://github.com/cflint/CFLint/issues/499.  I think it is fixed now.
